spark/core.py

```diff
@@ -333,6 +333,12 @@ def background(self, *args): pass
 
     @extern
     def rotate(self, *args): pass
+    
+    @extern
+    def translate(self, *args): pass
+    
+    @extern
+    def scale(self, *args): pass
```

Comment:
I recommend flipping the order
```suggestion
    @extern
    def translate(self, *args): pass
    
    @extern
    def rotate(self, *args): pass
    
    @extern
    def scale(self, *args): pass
```